presto-main/src/main/java/com/facebook/presto/type/UnknownOperators.java

```diff
@@ -13,87 +13,312 @@
+
+        @Override
+        public ScalarFunctionImplementation specialize(BoundVariables boundVariables, int arity, TypeManager typeManager, FunctionRegistry functionRegistry)
+        {
+            List<TypeSignature> resolvedParameterTypeSignatures = bindVariables(getSignature().getArgumentTypes(), boundVariables);
+            List<Type> resolvedParameterTypes = resolveTypes(resolvedParameterTypeSignatures, typeManager);
+            TypeSignature resolvedReturnTypeSignature = bindVariables(getSignature().getReturnType(), boundVariables);
+            Type resolvedReturnType = typeManager.getType(resolvedReturnTypeSignature);
+
+            List<Class<?>> parametersJavaTypes = resolvedParameterTypes.stream()
+                    .map((prestoType) -> Primitives.wrap(prestoType.getJavaType()))
+                    .collect(Collectors.toList());
+            Class<?> returnJavaType = Primitives.wrap(resolvedReturnType.getJavaType());
+
+            ImplementationSignature implementationSignature = new ImplementationSignature(returnJavaType, parametersJavaTypes);
+            MethodHandle implementation = getFunctionImplementation(implementationSignature);
+
+            Boolean[] nullableArguments = new Boolean[resolvedParameterTypes.size()];
+            fill(nullableArguments, Boolean.TRUE);
+
+            return new ScalarFunctionImplementation(true, ImmutableList.copyOf(nullableArguments), implementation, true);
+        }
+
+        @Override
+        public boolean isHidden()
+        {
+            return true;
+        }
+
+        @Override
+        public boolean isDeterministic()
+        {
+            return true;
+        }
+
+        @Override
+        public String getDescription()
+        {
+            return "Unknown operator";
+        }
     }
 
-    @ScalarOperator(GREATER_THAN_OR_EQUAL)
-    @Nullable
-    @SqlType(StandardTypes.BOOLEAN)
-    public static Boolean greaterThanOrEqual(@SqlType("unknown") @Nullable Void left, @SqlType("unknown") @Nullable Void right)
+    private static Cache<ImplementationSignature, MethodHandle> implementations = CacheBuilder.newBuilder().build();
+
+    private static MethodHandle getFunctionImplementation(ImplementationSignature implementationSignature)
     {
-        return null;
+        try {
+            return implementations.get(implementationSignature, () -> generateFunctionImplementation(implementationSignature));
+        }
+        catch (ExecutionException e) {
+            throw Throwables.propagate(e.getCause());
+        }
     }
 
-    @ScalarOperator(BETWEEN)
-    @Nullable
-    @SqlType(StandardTypes.BOOLEAN)
-    public static Boolean between(@SqlType("unknown") @Nullable Void value, @SqlType("unknown") @Nullable Void min, @SqlType("unknown") @Nullable Void max)
+    private static MethodHandle generateFunctionImplementation(ImplementationSignature implementationSignature)
     {
-        return null;
+        ClassDefinition definition = new ClassDefinition(
```

Comment:
We should be able to do this entirely with method handles, since it just needs to return a constant null.